pkg/restore/conflicts.go

```diff
@@ -0,0 +1,52 @@
+/*
+Copyright 2018 the Heptio Ark contributors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package restore
+
+import (
+	"github.com/heptio/ark/pkg/util/collections"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"strings"
+)
+
+// mergeSecrets takes image pull secrets and secrets from a backed up service accounts and merges them into the current in-cluster service account
+// The default token secret from the backed up serviceaccount will be ignored in favor of the one already present
+func mergeSecrets(fromCluster, fromBackup *unstructured.Unstructured) (*unstructured.Unstructured, error) {
+	var desired *unstructured.Unstructured
+	desired = fromCluster.DeepCopy()
+	secrets, _ := collections.GetSlice(fromBackup.UnstructuredContent(), "secrets")
+	for i := len(secrets) - 1; i >= 0; i-- {
+		secret := secrets[i]
```

Comment:
don't really need to declare `secret`, can just refer to `secrets[i]` in the next line